internal/common/disasters/commonpooldefinition.go

```diff
@@ -0,0 +1,55 @@
+package disasters
+
+import (
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+type Commonpool struct {
+	Resource, Threshold  uint
+}
+
+//islandContribution takes the resource donation from island to common pool
+func (e Environment) islandContribution(resource uint) {
+	e.CommonPool.Resource = resource
```

Comment:
should this be += ?

Reply:
and should it also be `func (e *Environment)`? this doesn't do anything.

See: https://tour.golang.org/methods/4

Reply:
Oh ok, this is unused. just a note though.